tests/test.js

```diff
@@ -1,7 +1,5 @@
 (function() {
 
-var mixpanel; // don't use window.mixpanel, use instance passed to test_mixpanel()
```

Comment:
did something happen in rebase-land? I'm pretty sure this is going to mess up test scenarios for module-based builds (anything which doesn't create `window.mixpanel`)